components/StandardListItem/StandardListItem.jsx

```diff
@@ -0,0 +1,46 @@
+import {PropTypes, Component } from 'react'
+import React from 'react'
+
+require('./StandardListItemStyles.scss')
+ // showIcon: true -> render the icon
+ // showLabel: true -> render the label
+ // imgSrc: source for the icon
+ // labelText: The text for the label
+ // placeIcon: defines the position of the icon. Either: top | left | right.  Default to top
+
+class StandardListItem extends Component {
```

Comment:
Components without state should be functional components. See https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html under "Stateless functional components"